integration-test/integration-test.ps1

```diff
@@ -0,0 +1,287 @@
+﻿function Set-LocalEnvironmentVariable {
+<#
+.SYNOPSIS
+	Set enviornment variable for this script scope
+.DESCRIPTION
+	This PowerShell function set the eviornment variable for this script.
+#>
+    param (
+        [Parameter()]
+        [System.String]
+        $Name,
+
+        [Parameter()]
+        [System.String]
+        $Value,
+
+        [Parameter()]
+        [Switch]
+        $Append
+    )
+    if($Append.IsPresent)
+    {
+        if(Test-Path "env:$Name")
+        {
+            $Value = (Get-Item "env:$Name").Value + $Value
+        }
+    }
+    Set-Item env:$Name -Value "$value" | Out-Null
+}
+
+function Get-LocalEnviornmentVariable {
+<#
+.SYNOPSIS
+	Get enviornment variable for this script scope
+.DESCRIPTION
+	This PowerShell function get the eviornment variable for this script.
+#>
+    param (
+        [Parameter()]
+        [System.String]
+        $Name
+    )
+    $ValVar = $null
+    if(Test-Path "env:$Name") {
+        $ValVar = (Get-Item "env:$Name").Value
+    }
+    return $ValVar
+}
+
+Function Validate_Variables {
+<#
+.SYNOPSIS
+	Check required enviornment variable for this script execution scope
+.DESCRIPTION
+	This PowerShell function validates required variables and stop the execution of the script.
+#>
+    
+
+    #$global:FCLI_MODULE = Read-Host -Prompt $MODULE_INPUT_STR
+    #$FCLI_URL = Read-Host -Prompt "Enter FOD url, [Default] https://api.sandbox.fortify.com:"
+    #$FCLI_FOD_TENANT = Read-Host -Prompt "Enter FOD Tenant:"
+    #$FCLI_FOD_USER = Read-Host -Prompt "Enter FOD Username:"
+    #$FCLI_FOD_PWD = Read-Host -Prompt "Enter FOD Password:" -AsSecureString
+
+    $global:FCLI_MODULE = Get-LocalEnviornmentVariable "FCLI_MODULE"
+    if($global:FCLI_MODULE -eq $null -or $global:FCLI_MODULE -eq "") {
+        if ($global:blnLinux) {
+            $global:FCLI_MODULE = "fcli.jar"
+        } else {
+            $global:FCLI_MODULE = "fcli.exe"
+        }
+    }
+
+    if (-not(Test-Path -Path $global:FCLI_MODULE -PathType Leaf)) {
+        Write-Host "The file [$global:FCLI_MODULE] has not been found.set variable 'FCLI_MODULE' with exact path" -ForegroundColor Red
+        Break
+    }
+
+    if($FCLI_URL -eq $null -or $FCLI_URL -eq "") {
+        Set-LocalEnvironmentVariable "FCLI_DEFAULT_FOD_URL" "https://api.ams.fortify.com"
+    }else{
+        Write-Host "Please set a enviornment varibale for different FoD URL, the Default is set as 'ams.fortify.com'" -ForegroundColor Red
+        #Set-LocalEnvironmentVariable "FCLI_DEFAULT_FOD_URL" $FCLI_URL
+    }
+
+    if($FCLI_FOD_TENANT -ne $null -or $FCLI_FOD_TENANT -ne "") {
+        Set-LocalEnvironmentVariable "FCLI_DEFAULT_FOD_TENANT" $FCLI_FOD_TENANT

```

Comment:
If FCLI_FOD_TENANT is not set this sets FCLI_DEFAULT_FOD_TENANT to nothing:

if($FCLI_FOD_TENANT -ne $null -or $FCLI_FOD_TENANT -ne "") { Write-Host ">$FCLI_FOD_TENANT<" }
><

With Powershell shouldn't this just be:

 if ($FCLI_FOD_TENANT) { ...